changer-d-epoque.csl

```diff
@@ -0,0 +1,356 @@
+          <text variable="collection-number"/>
+        </group>
+      </group>
+    </macro>
+    <macro name="locator">
+      <group delimiter=" ">
+        <label variable="locator" form="short"/>
+        <text variable="locator"/>
+      </group>
+    </macro>
+    <macro name="locator-cit">
+      <choose>
+        <if type="article-journal article-magazine article-newspaper entry-dictionary entry-encyclopedia chapter paper-conference" match="any">
+          <group delimiter=" " prefix="en partic. ">
+            <label variable="locator" form="short"/>
+            <text variable="locator"/>
+          </group>
+        </if>
+        <else>
+          <text macro="locator"/>
+        </else>
+      </choose>
+    </macro>
+    <macro name="title">
+      <choose>
+        <if type="book" match="any">
+          <text variable="title" font-style="italic"/>
+        </if>
+        <else-if type="webpage" match="any">
+          <text variable="title"/>
+        </else-if>
+        <else-if variable="container-title" match="none">
+          <text variable="title" font-style="italic"/>
+        </else-if>
+        <else>
+          <text variable="title"/>
+        </else>
+      </choose>
+    </macro>
+    <macro name="publisher">
+      <group delimiter=", ">
+        <text variable="publisher-place"/>
+        <text variable="publisher"/>
+      </group>
+    </macro>
+    <macro name="year-date">
+      <choose>
+        <if variable="issued">
+          <date variable="issued">
+            <date-part name="year"/>
+          </date>
+        </if>
+        <else>
+          <text term="no date" form="short"/>
+        </else>
+      </choose>
+    </macro>
+    <macro name="edition">
+      <choose>
+        <if is-numeric="edition">
+          <group delimiter=" ">
+            <number variable="edition" form="ordinal"/>
+            <text term="edition" form="short" text-case="lowercase"/>
+          </group>
+        </if>
+        <else>
+          <text variable="edition"/>
+        </else>
+      </choose>
+    </macro>
+    <citation et-al-min="4" et-al-use-first="1" collapse="year">
+      <layout suffix="." delimiter=" ; ">
+        <choose>
+          <if position="ibid-with-locator">
+            <group delimiter=", ">
+              <text term="ibid" text-case="capitalize-first" font-style="italic"/>
+              <text macro="locator"/>
+            </group>
+          </if>
+          <else-if position="ibid">
+            <text term="ibid" text-case="capitalize-first" font-style="italic"/>
+          </else-if>
+          <else-if position="subsequent">
+            <group delimiter=", ">
+              <text macro="author"/>
+              <text variable="title" form="short" font-style="italic"/>
+              <group delimiter=" ">
+                <text term="cited" font-style="italic"/>
+                <text value="supra" font-style="italic"/>
+                <text term="note" form="short" suffix=" $$"/>
```

Comment:
oh sorry, I read that as variable, not term, so that makes more sense.
But yes, `first-reference-note-number`  works well with Zotero or Mendeley and word (doesn't work in Pandoc). 
There's nothing special to it, you just use it like any other variable, i.e. something like

```
<group delimiter=" ">
  <text term="note" form="short"/>
  <text variable="first-reference-note-number"/>
</group>

will give you "n. 23" (assuming this was first cited in the 23rd footnote, obviously).